src/script/lua_api/l_env.h

```diff
@@ -116,7 +116,7 @@ class ModApiEnvMod : public ModApiBase {
 	// get_day_count() -> int
 	static int l_get_day_count(lua_State *L);
 
-	// find_node_near(pos, radius, nodenames) -> pos or nil
+	// find_node_near(pos, radius, nodenames, search_centre) -> pos or nil
```

Comment:
Although i think UK English spelling should be used in MT unfortunately there is a lot of 'center' 'normalize' USA English in MT and it may be the majority, i would go with what is mostly used.